app/models/manageiq/providers/vmware/infra_manager/storage.rb

```diff
@@ -1,3 +1,11 @@
 class ManageIQ::Providers::Vmware::InfraManager::Storage < ManageIQ::Providers::InfraManager::Storage
   include ManageIQ::Providers::Vmware::InfraManager::EmsRefObjMixin
+
+  supports :smartstate_analysis do
+    if nil?
+      unsupported_reason_add(:smartstate_analysis, "Storage not found")
+    elsif !storage_type_supported_for_ssa?
+      unsupported_reason_add(:smartstate_analysis, "Smartstate Analysis unsupported for storage type %{store_type}" % {:store_type => store_type})
+    end
+  end
```

Comment:
I don't think we need this, storage smartstate is different and the store_type shouldn't matter (plus `if nil?` doesn't make sense)